src/set_theory/surreal/basic.lean

```diff
@@ -117,50 +117,15 @@ lt_of_lf o (o.move_right j) (pgame.lf_move_right j)
 theorem numeric.le_move_right {x : pgame} (o : numeric x) (j) : x ≤ x.move_right j :=
 (o.lt_move_right j).le
 
--- TODO: this can be generalized to `add_lf_add_of_lf_of_lt`, which doesn't depend on any `numeric`
--- hypotheses.
-theorem add_lf_add
-  {w x y z : pgame.{u}} (oy : numeric y) (oz : numeric z)
-  (hwx : w ⧏ x) (hyz : y ⧏ z) : w + y ⧏ x + z :=
```

Comment:
For reviewers and maintainers: the way to recover this is to deduce `y < z` from `y ⧏ z`, `numeric y` and `numeric z`, and then `y ≤ z` follows, so we can apply `add_lf_add_of_lf_of_le`.

Reply:
Could we have this in this PR, then? It should not be removed, even if it is only a few steps from the new statement.

Reply:
I don't believe we should have this theorem. I had stated previously that, since `⧏` implies `<` for numeric games, and `<` implies `⧏` always, all the lemmas on numeric games should use `<` instead of `⧏`.

Reply:
I agree that it makes little sense to talk about ⧏ between numeric games, so the correct way to state this is to replace `hyz : y ⧏ z` by `y < z`. But then this holds unconditionally and could be called `add_lf_add_of_lf_of_lt`. But we already have `add_lf_add_of_lf_of_le` in this PR, and it also makes little sense to duplicate every lemma just to replace the `le` hypothesis by the stronger `lt`.

Reply:
`add_lf_add_of_lf_of_lt` is a weaker lemma than `add_lf_add_of_lf_of_le` though, the latter immediately implies the former. So the one I'm PRing is the one we should have.